proxylib/cassandra/cassandraparser_test.go

```diff
@@ -0,0 +1,282 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cassandra
+
+import (
+	"encoding/hex"
+	"testing"
+
+	// "github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/proxylib/accesslog"
+	"github.com/cilium/cilium/proxylib/proxylib"
+	"github.com/cilium/cilium/proxylib/test"
+
+	// log "github.com/sirupsen/logrus"
```

Comment:
Leftover?

Reply:
Yes and no. Toggling on the debug log level when developing new tests requires these lines. My preference is to keep these as comments rather than re-insert them every time new tests are added.